inspirehep/records/api/base.py

```diff
@@ -250,12 +275,28 @@ def create(cls, data, **kwargs):
             record = super().create(data, id_=id_, **kwargs)
         return record
 
-    @classmethod
-    def get_linked_records_in_field(cls, data, path):
+    def get_records_pid_from_field(self, path):
+        """Returns list of tuples (pid_type, pid_value) of all linked records in field
+        from specified path
+        Args:
+            path (str): the path of the linked records.
```

Comment:
maybe explain that `path` needs to be understood as a `get_value` argument? we probably need a specific name for this thing.